Public/Src/Engine/Processes/SandboxedProcessPipExecutor.cs

```diff
@@ -1815,7 +1828,7 @@ private bool PrepareFileAccessMonitoringCommon(LoggingContext loggingContext)
             if (allowInternalErrorsLogging || checkMessageCount)
             {
                 // Create unique file name.
-                m_detoursFailuresFile = GetDetoursInternalErrorFilePath(loggingContext);
+                m_detoursFailuresFile = GetDetoursInternalErrorFilePath(m_loggingContext);

```

Comment:
Do we need to pass a logging context here? Can the method simply access `m_loggingContext` ?